bmcremedy_connector.py

```diff
@@ -402,6 +469,56 @@ def _get_url(self, action_result, incident_number):
 
         return phantom.APP_SUCCESS, url
 
+    def _bmc_int_auth_refresh(self, action_result):
+
+        self.debug_print("In bmc_int_auth_refresh method")
+
+        config = self.get_config()
+        client_id, client_secret = config.get('client_id'), config.get('client_secret')
+        oauth_token = self._state.get('oauth_token')
+
+        if not (oauth_token and oauth_token.get("refresh_token")):
+            self._reset_the_state()
+            action_result.set_status(phantom.APP_ERROR, "Unable to get refresh token. Please run Test Connectivity again")
```

Comment:
There are a lot of duplicated messages here and below. Can we move these to consts to keep consistency?

Reply:
Will add message in consts.